.github/workflows/publish.yml

```diff
@@ -61,10 +61,6 @@ jobs:
           git fetch origin dist
           git checkout -b dist origin/dist
 
-      - name: Cleanup old /latest folder
```

Comment:
Removing this as I realised it won't actually work as expected here - being called here it would remove the newly generated version.

I think this is realistically an edge case, as the main schema file which exists here to be accessed by the `/latest` URL will be stable.